registry_module.go

```diff
@@ -117,6 +129,19 @@ type RegistryModuleID struct {
 	RegistryName RegistryName
 }
 
+// RegistryModuleDeleteByNameOrProvider represents the values requires to delete the entire registry module
+type RegistryModuleDeleteByNameOrProvider struct {
+	// The organization the module belongs to, see RegistryModule.Organization.Name
+	Organization string
+	// The name of the module, see RegistryModule.Name
+	Name string
+	// The namespace of the module. For private modules this is the name of the organization that owns the module
+	// Namespace is required for public modules
```

Comment:
It seems like both `Namespace` and `RegistryName` is always required for the deleteByNameorProvider to succeed. Can you delete the private registry by name or provider without these params?

Reply:
Correct, I will update the comment.